### PR TITLE
Data API for genomes

### PR DESCRIPTION
**Pull request by mlhenderson**

### CODE REVIEW

File: Proposed product descriptions/Data_API_for_Genomes_product_description.md [hunk 1]
```diff
@@ -0,0 +1,395 @@
...
+
+-   Data type refactoring and normalization
+
+-   Unified set of accessors to both new (refactored) and old (legacy) types
+
+-   Capability to create/upload data for the new types
+
+-   Client/server deployment
+
+-   Support for languages: Python, JavaScript, Perl
```
Comment (MrCreosote) [inline Proposed product descriptions/Data_API_for_Genomes_product_description.md:50]: No Java?

**Comment by nlharris** [comment]
Which proposed campaign does this correspond to on https://docs.google.com/spreadsheets/d/1HpmGSwvssS4ZfgsF3EHmWi9kITHIi5Qq1TqkEVMRNRE/edit#gid=0 ?

File: Proposed product descriptions/Data_API_for_Genomes_product_description.md (hunk 1 above)
Comment (mlhenderson) [inline Proposed product descriptions/Data_API_for_Genomes_product_description.md:50]: This is a representation of work done and essentially why, not the rainbow
of future possibilities.  Of course we would want to support Java as well.

**Comment by mlhenderson** [comment]
This is not a Q2 proposal.  I was told Data API did not have a PD, so I am
submitting under proposals.  I can submit elsewhere if someone tells me
where that should be.

**Comment by nlharris** [comment]
Ah, ok, right.

**Pull request merged**